package.json

```diff
@@ -36,8 +36,11 @@
     "async": "^1.5.2",
     "buffer-equal": "^1.0.0",
     "cesium": "^1.17.0",
+    "clone": "^1.0.2",
```

Comment:
Actually, it looks like `Cesium.clone` when I try to deep clone the gltf object with loaded sources, because the Node Buffer class doesn't have a default constructor. Is there a workaround for this?